### PR TITLE
[zh]resync k8s.io/core/v1/service for kubernetes API

### PR DESCRIPTION
**Pull request by howieyuen**
I have intended to read the explanation of field `spec.ports.targetPort` just now,  but it was missing, so I resync the whole page and found some nits.

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 1]
```diff
@@ -119,9 +119,12 @@ ServiceSpec 描述用户在服务上创建的属性。
...
 
-    <!-- 
-    Number or name of the port to access on the Pods targeted by the service. Number must be in the range 1 to 65535. Name must be an IANA_SVC_NAME. If this is a string, it will be looked up as a named port in the target Pod's container ports. If this is not specified, the value of the 'port' field is used (an identity map). This field is ignored for services with clusterIP=None, and should be omitted or set equal to the 'port' field. More info: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
-    -->
+  <!--
+  - **ports.targetPort** (IntOrString)
+
+    Number or name of the port to access on the pods targeted by the service. Number must be in the range 1 to 65535. Name must be an IANA_SVC_NAME. If this is a string, it will be looked up as a named port in the target Pod's container ports. If this is not specified, the value of the 'port' field is used (an identity map). This field is ignored for services with clusterIP=None, and should be omitted or set equal to the 'port' field. More info: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
+  -->
+  - **ports.targetPort** (IntOrString)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:127]: 127 行前要加空行

**Comment by tengqm** [comment]
"watch" 统一翻译成 “监测” 好一些，本页中翻译成“观察”了

**Comment by howieyuen** [comment]
> "watch" 统一翻译成 “监测” 好一些，本页中翻译成“观察”了

我也看到了，犹豫了一下。现已改正。

**Pull request merged**